packages/context/src/api/paramsSerializer.ts

```diff
@@ -1,11 +1,24 @@
 import { ParamsSerializerOptions } from 'axios';
+import _ from 'lodash';
```

Comment:
Don't use lodash for browser.  lodash-es allows webpack to do tree shaking, though having said that, there's no reason to use lodash to check if something is undefined.